.github/workflows/pkg-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: 'pkg-pr'
+on:
+  push:
+    branches:
+      - 'refactor/node'
+    tags:
+      - '!**'
+  pull_request:
+    branches:
+      - 'refactor/node'
+  pull_request_review:
+    types: [submitted]
+
+
+jobs:
+  continuous-release:
+    if: github.repository == 'farm-fe/farm' &&  github.event.review.state == 'APPROVED'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Cache rust artifacts
+        uses: oxidecomputer/actions-rs_toolchain@oxide/master
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+
```

Comment:
Action failed here https://github.com/farm-fe/farm/actions/runs/11178171789/job/31075215488?pr=1801, a `version` field should exsits. Perhaps try with `version: 9`?